js/ConfidenceSliderModel.js

```diff
@@ -130,7 +132,9 @@ export default class ConfidenceSliderModel extends SliderModel {
   }
 
   _setupLinkedModel() {
-    this.linkedModel = Adapt.components.findWhere({ _id: this.get('_linkedToId') });
+    const linkedToId = this.get('_linkedToId');
+    if (!linkedToId) return;
+    this.linkedModel = Adapt.components.findWhere({ _id: linkedToId });
```

Comment:
```suggestion
    this.linkedModel = data.findById(linkedToId);
```

Reply:
This was the code already in use, but can amend accordingly.